src/Collector/CollectorInterface.php

```diff
@@ -2,30 +2,48 @@
 
 namespace SensioLabs\Deptrac\Collector;
 
+use SensioLabs\AstRunner\AstParser\AstClassReferenceInterface;
 use SensioLabs\AstRunner\AstParser\AstParserInterface;
-use SensioLabs\Deptrac\CollectorFactory;
 use SensioLabs\AstRunner\AstMap;
-use SensioLabs\AstRunner\AstParser\AstClassReferenceInterface;
+use SensioLabs\Deptrac\CollectorFactory;
 
+/**
+ * A collector is responsible to tell from an AST node (e.g. a specific class) is part of a layer.
+ */
 interface CollectorInterface
```

Comment:
Added some doc to the interface, I personally tend to prefer self-explaining and documented doc although I appreciate some higher level doc as well.